glide.lock

```diff
@@ -1,5 +1,5 @@
 hash: e2b8455d2f47652e91d0350ad7994f4824c6728af81e95c454094734bccbba32
```

Comment:
`glide.lock` needs to be updated to reflect changes to dependencies: not importing snap, importing lib-go. I suggest removing both `glide.lock`, `glide.yaml`, clearing glide cache with `$ glide cc` and then going through `$ glide init` to ensure proper dependency list.